src/app/store/route3/state.ts

```diff
@@ -0,0 +1,16 @@
+import { Route3Dex } from 'lib/apis/route3/fetch-route3-dexes';
+import { Route3SwapParamsResponse } from 'lib/apis/route3/fetch-route3-swap-params';
+import { Route3Token } from 'lib/apis/route3/fetch-route3-tokens';
+import { createEntity, LoadableEntityState } from 'lib/store';
+
+export interface Route3State {
```

Comment:
I think it's better to call this `SwapState`, as this's a feature name, not cpecific implementation.